src/config.cpp

```diff
@@ -243,8 +252,16 @@ Config pkgi_load_config()
                 config.no_version_check = 1;
             else if (pkgi_stricmp(key, "install_psp_as_pbp") == 0)
                 config.install_psp_as_pbp = 1;
+            else if (pkgi_stricmp(key, "install_psv_location") == 0)
+                config.install_psv_location = value;
```

Comment:
It's fine to have the Vita install location as part of the config struct, but don't allow the user to set it to anything besides the default.

Reply:
Understood. I've adjusted both locations.